lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -83,10 +89,18 @@
                     var info = {
                         alreadyMember: false,
                         memberOfCurrentTeam: false,
-                        teamHasSpace: false
+                        teamHasSpace: false,
+                        isAdminOrStaff: false,
+                        isInstructorManagedTopic: false
                     };
 
                     info.memberOfCurrentTeam = TeamUtils.isUserMemberOfTeam(this.model.get('membership'), username);
+                    info.isAdminOrStaff = this.context.userInfo.privileged || this.context.userInfo.staff;
+
+                    if (this.topic.attributes.type !== undefined) {
```

Comment:
This is the same logic as in `team_profile.js`. Please refactor so this same logic is put into a common function.